zerver/lib/mime_types.py

```diff
@@ -36,6 +36,7 @@
     "image/jpeg",
     "image/png",
     "image/webp",
+    "image/svg+xml",
```

Comment:
Just a few lines below this, we have the following comment:
```python
    # To avoid cross-site scripting attacks, DO NOT add types such
    # as application/xhtml+xml, application/x-shockwave-flash,
    # image/svg+xml, text/html, or text/xml.
```

I don't know how to make that comment more clear.